doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
+#########################
+
+Many packages in Refkit come with a default set of configuration files.
+This document gives guidance for configuring a Refkit image with custom
+package configuration files.
+
+Using .bbappend files to override package configuration
+=======================================================
+
+If you have only a limited set of packages which you need to configure,
+you can create ``.bbappend`` files which append the recipes for each
+package which you want to modify. A usual way to change the
+configuration is to change the install phase to replace or remove the
+default configuration files.
+
+As explained in `yocto-compatible.rst <yocto-compatible.rst>`_, the
```

Comment:
Ah. I was under the impression that all Yocto layers (including the custom distro configuration layer) are expected to be "Yocto Compatible" soon, and so including them would be expected not to change Refkit configuration as such. Does this depend on whether the customization layer is added inside refkit or if refkit is added inside the customization layer (meta-oe style)?